### PR TITLE
Insilico reports

### PR DESCRIPTION
**Pull request by alvaralmstedt**
Added version tag and computed sex to main qc report. Added flag for skipping petagene compression. Added insilico coverage reports for all 3 options (tn no to). Normals sample will be used for insilico reports if available.

TODO: Merge insilico coverage reports into main qc excel.

### CODE REVIEW

File: configs/cluster.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 __default__:
-    queue: wgs.q
+    queue: production.q
```
Comment (hannasoderstrom1) [inline configs/cluster.yaml:2]: why production queue?

File: configs/insilico_config.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "ChiCap_Actionable_v2.0": {
+        "bedfile": "ChiCap_Actionable.bed",
```
Comment (hannasoderstrom1) [inline configs/insilico_config.json:3]: where are the bedfiles located? in the repo?

File: pipeline.snakefile [hunk 3]
```diff
@@ -193,14 +201,20 @@ def upload_somatic_iva(wildcards):
...
+
 rule all:
     input: 
-        get_igv_input,
-        upload_somatic_iva,
-        upload_germline_iva,
+        #get_igv_input,
         expand("{workingdir}/reporting/workflow_finished.txt", workingdir=workingdir),
-        alissa_vcf_conversion
+        #alissa_vcf_conversion
```
Comment (hannasoderstrom1) [inline pipeline.snakefile:220]: does the alissa vcf conversion still work?

File: wgs_somatic-run-wrapper.py [hunk 4]
```diff
@@ -110,6 +113,66 @@ def analysis_end(outputdir, tumorsample, normalsample):
     else:
         pass
 
+
+def get_insilico_info(all_insilico=True, panel_names=[]):
```
Comment (hannasoderstrom1) [inline wgs_somatic-run-wrapper.py:117]: why are these insilico functions in the wrapper script? they are not used in the wrapper?

File: workflows/scripts/insilico_coverage/anybelow_sample.py [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                writer.writerow({"chr": str(i["chr"]),
+                                 "start": str(i["start"]),
+                                 "stop": str(i["stop"]),
+                                 "gene": str(i["gene"]),
+                                 "transcript": str(i["transcript"]),
+                                 "exon": str(i["exon"])})
+    return outfile
+
+
+#Function to be run from other scripts
```
Comment (hannasoderstrom1) [inline workflows/scripts/insilico_coverage/anybelow_sample.py:78]: what does this function do except from "run from other scripts"?

File: workflows/scripts/insilico_coverage/anybelow_sample.py [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+def run(infile,  limits=[20, 10]):
+    outs = []
+    for i in limits:
+        outs.append(merge_adjacent(read_input(infile, i)))
+    return outs
+
+
+def alt_main(csv_input, csv_output_10x, csv_output_20x):
```
Comment (hannasoderstrom1) [inline workflows/scripts/insilico_coverage/anybelow_sample.py:86]: what is this function used for?

File: workflows/scripts/insilico_coverage/stats_for_pdf.py [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    pass
+
+def get_pct_cov():
+    pass
+
+def genes_and_transcripts():
+    pass
+
+def get_regions_covered_below():
+    pass
```
Comment (hannasoderstrom1) [inline workflows/scripts/insilico_coverage/stats_for_pdf.py:16]: these functions don't do anything?

**Review by hannasoderstrom1 — CHANGES_REQUESTED** [review]
added some comments

File: workflows/scripts/insilico_coverage/anybelow_sample.py (hunk 5 above)
Comment (alvaralmstedt) [reply workflows/scripts/insilico_coverage/anybelow_sample.py:78]: Vestigial function from WOPR. I assume it is for if you need to rerun this part separately from the pipeline

File: configs/cluster.yaml (hunk 1 above)
Comment (hannasoderstrom1) [reply configs/cluster.yaml:2]: change it back to wgs queue before merging

File: workflows/scripts/insilico_coverage/anybelow_sample.py (hunk 6 above)
Comment (alvaralmstedt) [reply workflows/scripts/insilico_coverage/anybelow_sample.py:86]: Who knows. I don't dare to remove it at the moment :)

Comment (alvaralmstedt) [reply workflows/scripts/insilico_coverage/anybelow_sample.py:86]: Well it generates some csv files that are used for the insilico reports

File: workflows/scripts/insilico_coverage/stats_for_pdf.py (hunk 7 above)
Comment (alvaralmstedt) [reply workflows/scripts/insilico_coverage/stats_for_pdf.py:16]: damn right they dont

**Review by alvaralmstedt — COMMENTED** [review]
wanna merge

**Pull request merged**